crowdkit/aggregation/classification/glad.py

```diff
@@ -23,13 +23,12 @@
 
 @attr.s
 class GLAD(BaseClassificationAggregator):
-    r"""Generative model of Labels, Abilities, and Difficulties.
+    r"""The $\boldsymbol{GLAD}$ (Generative model of Labels, Abilities, and Difficulties) model is a probabilistic model that parametrizes the abilities of workers and the difficulty of tasks.
```

Comment:
What's the difference between just` **bold**` and `$\boldsymbol{bold}$`?

Reply:
As I understood, the `$\boldsymbol{bold}$` is used in formulas. In other cases, the `**bold**` is needed. I fixed it in all files